### PR TITLE
Update docs to reflect new idToken parameter available in reconcile

### PR DESCRIPTION
**Pull request by robfusion**
https://github.com/FusionAuth/fusionauth-issues/issues/323

### CODE REVIEW

File: site/docs/v1/tech/lambdas/openid-connect-response-reconcile.adoc [hunk 1]
```diff
@@ -26,8 +26,9 @@ This lambda must contain a function named `reconcile` that takes three parameter
 * `user` - the FusionAuth User object
 * `registration` - the FusionAuth UserRegistration object
 * `jwt` - the JSON payload returned from the OpenID Connect Userinfo endpoint
+* `idToken (optional)` - the JSON id_token returned from the OpenID Connect Token endpoint. It will only be available if it can be validated using the client secret.
```
Comment (robotdan) [inline site/docs/v1/tech/lambdas/openid-connect-response-reconcile.adoc:29]: If provided, and can be validated

File: site/docs/v1/tech/lambdas/openid-connect-response-reconcile.adoc [hunk 2]
```diff
@@ -45,12 +46,22 @@ If the JWT from the OIDC identity provider does not come back with an email clai
...
-  user.email = jwt.sub + '@no-email-present.example.com';
+function(user, registration, jwt, idToken) {
+  // OR you can possibly get it from the idToken if available
+  if (idToken && idToken.email) {
+    console.info('email from id_token: ' + idToken.email);
+    user.email = idToken.email;
+  }
+  else {
+    // The user's unique Id is the 'sub' claim.
+    user.email = jwt.sub + '@no-email-present.example.com';
```
Comment (robotdan) [inline site/docs/v1/tech/lambdas/openid-connect-response-reconcile.adoc:62]: We actually no longer allow you to modify the `email` in a lambda. We should double check that, but as far as I know this won't work. We should come up with a new "example" lambda that does not modify the `email`.

**Comment by robotdan** [comment]
@robfusion sorry - not sure if I dropped the ball on this one. Is this all ready to go and to be merged?

**Comment by robfusion** [comment]
> @robfusion sorry - not sure if I dropped the ball on this one. Is this all ready to go and to be merged?

I think so. We definitely include the id token now so I believe it's correct.

**Comment by mooreds** [comment]
Can we close this out? I think I handled in https://github.com/FusionAuth/fusionauth-site/pull/1114 

@robfusion ?

**Comment by robotdan** [comment]
@robfusion can this one be closed out?

**Pull request closed without merging**